src/Modules/Meetings/Domain/MeetingComments/Events/MeetingCommentCreatedDomainEvent.cs

```diff
@@ -7,9 +7,15 @@ public class MeetingCommentCreatedDomainEvent : DomainEventBase
     {
         public MeetingCommentId MeetingCommentId { get; }
 
-        public MeetingCommentCreatedDomainEvent(MeetingCommentId meetingCommentId)
+        public MeetingCommentId? InReplyToCommentId { get; }
```

Comment:
I thought about create another event like `CommentReplyCreatedDomainEvent` instead to extend existed MeetingCommentCreatedDomainEvent. But I decided we don't have such domain entity as 'reply' and it would be strange if MeetingComment's constructor would generate two different events in different cases. @kgrzybek what do you think about the implementation?

Reply:
It depends on requirements but generally I would like to distinguish adding a new comment from adding a reply. So I would suggest renaming to `MeetingCommentAddedDomainEvent` and `ReplyToMeetingCommentAddedDomainEvent`

Reply:
Done.